doc/sphinx/source/users-guide/base/simpleConsoleOutput.rst

```diff
@@ -0,0 +1,44 @@
+:title: Users Guide: Simple Console Output: Hello world
+
+Simple Console Output: Hello world
+==================================
+
+A good feature to learn first in any language is how to print things
+out to the console.  Although Chapel has a sophisticated set of I/O
+features, let's just focus on a simple capability to get started.
+
+
+The ``writeln()`` routine prints its list of comma-separated
```

Comment:
I think that's a fair point.  Historically, we've used 'function' to refer to the union of 'procedures' and 'iterators', and in this treatment of the language, I'd also like to be able to use the term 'routine' for the same role, but also don't object to using 'procedure' here to be crisp (even though the user won't see that keyword for many more sections now).